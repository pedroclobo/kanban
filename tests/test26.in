u CWU1UYcuuZXhwjbI6sV0
u k2ClgGZIUNW8RWP5P3XS
u HKprNXjqbsLkkVTtubSd
u SlxOrPmv9GrFXZKH1UB5
u BECLJ0NIUgMH7sCgObD1
u DyA_YWc8iWsXQTkObnqk
u a86KxSDLq7JrY1l2Dpxx
u MZnTdymRfeok4esu0xZd
u s1XZL29fc14e5fgEP_y1
u Myic0GwLQn4W1lIWROUx
u VOtXMf_CxMFelrL9l_rd
u 1cxtXhwZgELm8uiBMXmd
u bVtbYFz4CCxIxjSngRAT
u e16xbzJf_Lo2DSW0PixS
u OTh26dxKMbU0A6gTpu8t
u UDBGCSsq48KdHHMmtsWo
u cTR8rxrH_RJv8vusIRpj
u uquo84npdvTeAer556lS
u iIg3CRl2vjP3xvX5HB_z
u 1ihGhh8rw26Q7n80XUHI
u O_BdRAwyJiHX0UPuI9Bv
u syRf3jNWIBwEcpCtwEuh
u D8u1wMIGUNhB6LQyo7ma
u VQZmiGOgHR94UH69lMsc
u nbN3g9aMRx0odm9Zn1Lj
u eW4mDorVPE3U0H1OcsHr
u ciXaUoon1ZJfIT_lvinE
u 90w9lZKBDhgP2xhJXq51
u Xo9NsZnsL7NYyx6lXrJb
u tMhrlKq6dvw4ORQnxxlB
u pJLSL11XWRa6qVWUZC2s
u trnJzekuSCXUTPx5bBtu
u ZEwpK_INR8Bvsgaw82zF
u h_GdFSKJtYurofb5itgH
u VDVeGC2WNNWrJWUVicne
u cuwz8QT98eY52ISR5nCf
u VumydBuT5M5_xfSqi_R_
u rXHJfyzDKqxvkzV8QVZh
u ZOnQ2tIn2Jfp12U4NtQr
u KzbmrKQphoSM0xJezKDE
u elSifpEMKN3K6_7Csh4b
u rbtLHn4IhwLGhiAPOQRm
u jJRHrLl0PgYwN_Xd97zu
u ruKr1h6oomp5YwTA1Nhq
u p_S_2QwY4HegxMx376KZ
u vozQYTHY_iGzclnhxhhr
u xdsshjwu_f3D917O6iZZ
u Ikk3Aw8ZpZPY3CJSNWxH
u vZC3vebxq9Je_94ihInR
u T0M7tYy8mulumYzqS_YE
u
q
