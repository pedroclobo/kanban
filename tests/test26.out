CWU1UYcuuZXhwjbI6sV0
k2ClgGZIUNW8RWP5P3XS
HKprNXjqbsLkkVTtubSd
SlxOrPmv9GrFXZKH1UB5
BECLJ0NIUgMH7sCgObD1
DyA_YWc8iWsXQTkObnqk
a86KxSDLq7JrY1l2Dpxx
MZnTdymRfeok4esu0xZd
s1XZL29fc14e5fgEP_y1
Myic0GwLQn4W1lIWROUx
VOtXMf_CxMFelrL9l_rd
1cxtXhwZgELm8uiBMXmd
bVtbYFz4CCxIxjSngRAT
e16xbzJf_Lo2DSW0PixS
OTh26dxKMbU0A6gTpu8t
UDBGCSsq48KdHHMmtsWo
cTR8rxrH_RJv8vusIRpj
uquo84npdvTeAer556lS
iIg3CRl2vjP3xvX5HB_z
1ihGhh8rw26Q7n80XUHI
O_BdRAwyJiHX0UPuI9Bv
syRf3jNWIBwEcpCtwEuh
D8u1wMIGUNhB6LQyo7ma
VQZmiGOgHR94UH69lMsc
nbN3g9aMRx0odm9Zn1Lj
eW4mDorVPE3U0H1OcsHr
ciXaUoon1ZJfIT_lvinE
90w9lZKBDhgP2xhJXq51
Xo9NsZnsL7NYyx6lXrJb
tMhrlKq6dvw4ORQnxxlB
pJLSL11XWRa6qVWUZC2s
trnJzekuSCXUTPx5bBtu
ZEwpK_INR8Bvsgaw82zF
h_GdFSKJtYurofb5itgH
VDVeGC2WNNWrJWUVicne
cuwz8QT98eY52ISR5nCf
VumydBuT5M5_xfSqi_R_
rXHJfyzDKqxvkzV8QVZh
ZOnQ2tIn2Jfp12U4NtQr
KzbmrKQphoSM0xJezKDE
elSifpEMKN3K6_7Csh4b
rbtLHn4IhwLGhiAPOQRm
jJRHrLl0PgYwN_Xd97zu
ruKr1h6oomp5YwTA1Nhq
p_S_2QwY4HegxMx376KZ
vozQYTHY_iGzclnhxhhr
xdsshjwu_f3D917O6iZZ
Ikk3Aw8ZpZPY3CJSNWxH
vZC3vebxq9Je_94ihInR
T0M7tYy8mulumYzqS_YE
